Dockerfile

```diff
@@ -11,6 +11,9 @@ ENV SUBSCRIPTIONS_PER_NEWSFEED="1024"
 ENV NEWSFEED_ID_LENGTH="128"
 ENV PROCESSOR_CONCURRENCY="4"
 
+ENV REDIS_DSN="redis://redis/0"
+ENV REDIS_CONN_TIMEOUT=5
```

Comment:
It would be great to use less shortcuts so I tend to ask to rename this to `REDIS_CONNECTION_TIMEOUT`.